zhaquirks/tuya/ts0601_valve.py

```diff
@@ -30,6 +43,22 @@ class TuyaValveTimerState(t.enum8):
     Enabled = 0x02
 
 
+class TuyaValveStatus(t.enum8):
+    """Tuya Irrigation Valve status enum."""
+
+    Off = 0x00
+    Auto = 0x01
+    Disabled = 0x02
+
+
+class TuyaValveWaterConsumedInstDemand(TuyaValveWaterConsumed):
+    """Tuya Valve Water consumed cluster, with instantaneous_demand."""
+
+    def __init__(self, *args, **kwargs):
+        """Init a TuyaValveWaterConsumed cluster."""
+        pass
```

Comment:
Hmm, I feel like it isn't that clean to override ``__init__`` just so the super class doesn't add the unsupported attribute.
I'd rather duplicate the entire class then (except for the unsupported attribute call) or change `TuyaValveWaterConsumed` so it doesn't do that and introduce another class that overrides ``__init__`` just to add that unsupported attribute and make that the default (or explicitly provide for all other valve quirks).

Reply:
Changed this around to be a bit cleaner.